godtools/App/Features/AppLanguage/Domain/Entities/LanguageDownloadStatusDomainModel.swift

```diff
@@ -13,4 +13,22 @@ enum LanguageDownloadStatusDomainModel: Equatable {
     case notDownloaded
     case downloading(progress: Double)
     case downloaded
+    
+    init(downloadedLanguage: DownloadedLanguageDataModel?) {
```

Comment:
Hey @rachaelblue one rule in the domain layer is it's not supposed to know anything about the data layer.  

However, it's perfectly fine that the data layer knows about the domain layer.  I'd suggest either performing this logic in the interface implementation, or you could even create an extension off of DownloadedLanguageDataModel.

Reply:
Ah good catch, still learning to think about which way the data is flowing.  Just moved into the interface implementation for now since the download status logic may change.